HACKING

```diff
@@ -32,6 +32,8 @@ Checklist for making a release:
 
  - Update the NEWS file
 
+ - Update the AUTHORS file if we have any new authors since last release.
```

Comment:
Please add the git command to create the list here. I don't want to google it for every release :-)